src/sentry/static/sentry/app/views/organizationDiscover/queryBuilder.jsx

```diff
@@ -193,6 +195,38 @@ export default function createQueryBuilder(initial = {}, organization) {
       });
   }
 
+  /**
+   * Get the actual query to be run for each visualization type
+   *
+   * @param {Object} originalQuery Original query input by user (external query representation)
+   * @param {String} Type to fetch - currently either byDay or base
+   * @returns {Object} Modified query to be run for that type
+   */
+  function getQueryByType(originalQuery, type) {
+    if (type === 'byDayQuery') {
+      return {
+        ...originalQuery,
+        groupby: ['time'],
+        rollup: 60 * 60 * 24,
+        orderby: 'time',
+        limit: 1000,
```

Comment:
What does this limit refer to  in this query?

Reply:
It's the largest number of results we can request from Snuba. Essentially if the user enters some lower number, we want to ignore that.